src/components/UiElements/SubscribeCard.js

```diff
@@ -10,11 +10,13 @@ import HighlightOffRoundedIcon from "@material-ui/icons/HighlightOffRounded";
 
 const useStyles = makeStyles((theme) => ({
   root: {
-    height: "330px",
-    display: "flex",
+    minHeight: "330px",
```

Comment:
Using 'px' for containers will not make the page responsive. Material UI grid classes are preferred